backend/api/src/http.rs

```diff
@@ -204,6 +204,9 @@ pub fn build(
             .configure(endpoints::media::configure)
             .configure(endpoints::session::configure)
             .configure(endpoints::locale::configure)
+            .configure(
+                |cfg| -> () { cfg.route("/", shared::api::Method::Get.route().to(no_content_response)); }
+            )
```

Comment:
prefer `service(<fn>)` and `#[actix_web::get("/")]` in this case- see [`get_spec`](https://github.com/ji-devs/ji-cloud/blob/05deb38f80c8d7d5aa8bb2703c01e2c170f0536a/backend/api/src/http.rs#L227-L232). Alternatively, use the following:
```rust
.route("/", paperclip::actix::web::get().to(<fn>))
```
Note, however, in this case, this can be succulently written with the following 1 liner:
```suggestion
            .route("/", paperclip::actix::web::get().to(|| NoContent))
```